resources/views/emails/training/waiting_list_atc_top_ten.blade.php

```diff
@@ -0,0 +1,25 @@
+@extends('emails.messages.post')
+
+@section('body')
+<p>
+    You have now reached the top 10 of the {{$list_name}} waiting list. You will soon be reaching the top of the list where providing you meet the eligibility criteria we will be offering you a training place. The criteria to be eligible for a training place can be found in the ATC Training Handbook sections 5.15.1 (S2 training), 5.16.1 (S3 training) and 5.17.1 (C1 training).
+</p>
+
+<p>
+    For all ratings you must have completed the relevant CTS theory exam to be eligible for a place. If you haven't already, we recommend that you start looking through the theory material now and when you feel ready you attempt the theory exam.
+</p>
+
+<p>
+    If you have not attempted the theory exam when you reach the top of the list, you may be removed from the waiting list and have to rejoin again at the start. This process is documented in the ATC Training Handbook section 5.4.3 which is included below.
+</p>
+
+<pre>
+        If you reach the top of the waiting list and a training place becomes available, but you have not yet passed the relevant CTS theory exam, you will be contacted by a member of the ATC Training staff. You will be given 7 days to attempt the theory exam. If unsuccessful, there is a 7 day cool down period, after which you will be given a further 8 days to complete the exam. This will continue until you pass the exam.
+
+        Should you not attempt the exam within the 7 days, you will be removed from the waiting list.
+</pre>
+
+<p>
+    If you have any issues with this and are still interested in training then reach out to us via the helpdesk with help topic 'Member Services'.
+</p>
+@stop
```

Comment:
I feel like this could do with some rewording. This should be an exciting email for our members to recieve to let them know that they'll soon be entering the world of training.

Reply:
This email content was provided by Adam, my only contribution here was correcting a typo that I spotted. 

Content writing certainly isn't a strong point of mine, so I'll stay well away and let Adam know about this feedback. 😀

Reply:
To be honest I didn't really think of it that way however I feel like exciting people when they reach top 10 could be counter-productive as 10th to 1st could be a matter of weeks or it could be a matter of months. The reason that we picked top 10 was to keep it conservative to ensure that everyone get plenty of notice to start working on the theoretical material.

I'll reword a little however it'll remain fairly similar to prevent people asking too many questions along the lines of "you said I'd have a training place soon and it's been 3 weeks"

Reply:
@1adamf I think my concern is that this is the first time some members may hear from us after 6-12 months and a lot of it centres around policy and how they could be removed. I know this is important information to convey but by making it a bit more "exciting" as I put it, may enthuse them to ensure they engage with the policy.